The test suite assume GNU diff and sed, and often hangs on the multiple jobs
test due to a race.

diff -wpruN '--exclude=*.orig' a~/Test/W02jobs.ztst a/Test/W02jobs.ztst
--- a~/Test/W02jobs.ztst	1970-01-01 00:00:00
+++ a/Test/W02jobs.ztst	1970-01-01 00:00:00
@@ -25,7 +25,7 @@
       # exit twice in case of check_jobs
       zpty -w zsh $'exit\nexit\n'
       # zpty gives no output when piped without these braces (?)
-      { zpty -r zsh } | sed $'/[^[:space:]]/!d; s/\r$//;'
+      { zpty -r zsh } | gsed $'/[^[:space:]]/!d; s/\r$//;'
       zpty -d
       :
     }
@@ -167,29 +167,29 @@
 *>\[1]  + (stopped|suspended)*sleep*
 # no 'SIGHUPed' message for suspended jobs
 
-  zpty_start
-  zpty_input 'sleep 10 & sleep 9 & sleep 8 & sleep 7 &'
-  zpty_line 4
-  zpty_input 'kill %4'
-  # Dunno why we need this when checking input...
-  sleep 0.1
-  zpty_line
-  zpty_input 'kill -HUP %3'
-  zpty_line
-  zpty_input 'kill -INT %2'
-  zpty_line
-  zpty_input 'kill -KILL %1'
-  zpty_line
-  zpty_stop
-0:various `kill` signals with multiple running jobs
-*>\[1] [0-9]##
-*>\[2] [0-9]##
-*>\[3] [0-9]##
-*>\[4] [0-9]##
-*>\[4]  ? terminate*sleep*
-*>\[3]  ? hangup*sleep*
-*>\[2]  ? interrupt*sleep*
-*>\[1]  ? kill*sleep*
+#  zpty_start
+#  zpty_input 'sleep 10 & sleep 9 & sleep 8 & sleep 7 &'
+#  zpty_line 4
+#  zpty_input 'kill %4'
+#  # Dunno why we need this when checking input...
+#  sleep 0.1
+#  zpty_line
+#  zpty_input 'kill -HUP %3'
+#  zpty_line
+#  zpty_input 'kill -INT %2'
+#  zpty_line
+#  zpty_input 'kill -KILL %1'
+#  zpty_line
+#  zpty_stop
+#0:various `kill` signals with multiple running jobs
+#*>\[1] [0-9]##
+#*>\[2] [0-9]##
+#*>\[3] [0-9]##
+#*>\[4] [0-9]##
+#*>\[4]  ? terminate*sleep*
+#*>\[3]  ? hangup*sleep*
+#*>\[2]  ? interrupt*sleep*
+#*>\[1]  ? kill*sleep*
 
 %clean
 
diff -wpruN '--exclude=*.orig' a~/Test/ztst.zsh a/Test/ztst.zsh
--- a~/Test/ztst.zsh	1970-01-01 00:00:00
+++ a/Test/ztst.zsh	1970-01-01 00:00:00
@@ -352,7 +352,7 @@ ZTST_diff() {
       diff_ret=1
     fi
   else
-    diff_out=$(diff -a "$@")
+    diff_out=$(gdiff -a "$@")
     diff_ret="$?"
     if [[ "$diff_ret" != "0" ]]; then
       print -r -- "$diff_out"
